api/PclusterApiHandler.py

```diff
@@ -33,6 +33,13 @@
 SECRET_ID = os.getenv("SECRET_ID")
 ENABLE_MFA = os.getenv("ENABLE_MFA")
 SITE_URL = os.getenv("SITE_URL", API_BASE_URL)
+SCOPES_LIST = os.getenv("SCOPES_LIST")
+REGION = os.getenv("AWS_DEFAULT_REGION")
+REDIRECT_URL = os.getenv("REDIRECT_URL", f"{SITE_URL}/login")
```

Comment:
REDIRECT_URL could just be `f"{SITE_URL}/login"` without the ENV var since we know where to redirect to